lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/FileBasedAccessControlConfig.java

```diff
@@ -14,31 +14,36 @@
 package io.trino.plugin.base.security;
 
 import io.airlift.configuration.Config;
-import io.airlift.configuration.validation.FileExists;
+import io.airlift.configuration.ConfigDescription;
 import io.airlift.units.Duration;
 import io.airlift.units.MinDuration;
 
 import javax.validation.constraints.NotNull;
 
-import java.io.File;
-
 public class FileBasedAccessControlConfig
 {
     public static final String SECURITY_CONFIG_FILE = "security.config-file";
     public static final String SECURITY_REFRESH_PERIOD = "security.refresh-period";
+    public static final String SECURITY_JSON_POINTER = "security.json-pointer";
 
-    private File configFile;
+    private String configFile;
     private Duration refreshPeriod;
+    private String jsonPointer = "";
+
+    public enum SecurityConfigSource
+    {
+        LOCAL,
+        REST;
+    }
```

Comment:
Can we derive them based on the `File` path

Reply:
enum removed